x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -92,9 +93,25 @@ const addRunningAgentToAgentPolicy = async (
   );
 };
 
+export const addPackagePolicyCspRules = async (
+  soClient: SavedObjectsClientContract,
+  packagePolicy: PackagePolicy
+): Promise<CspRulesStatus> => {
+  const rules = await getCspRules(soClient, packagePolicy);
```

Comment:
if you return aggregated data, you should not fetch all the rules in memory do the calculations yourself
this can be done using `aggs` in the query to the saved objects